src/main/java/uk/gov/hmcts/darts/audio/helper/TransformedMediaHelper.java

```diff
@@ -102,8 +102,12 @@ public TransformedMediaEntity createTransformedMediaEntity(MediaRequestEntity me
         if (nonNull(fileSize)) {
             entity.setOutputFilesize(fileSize.intValue());
         }
-        transformedMediaRepository.save(entity);
-        return entity;
+        //Ensures createdBy / LastModified does not get overridden by the @CreatedBy / @LastModifiedBy annotaiton
```

Comment:
I don't get this comment, why does doing a save ensure the created by or last modified does not get overridden? You have a typo in annotation as well.

Reply:
The updateLastModified function is a modifying query so it bypasses springs entity listener and thus the auto update of lastModifiedBy

Reply:
ok - you alright to fix the typo?

Reply:
Updated :)